src/python/py_tokenizer.cpp

```diff
@@ -30,10 +30,10 @@ void init_tokenizer(py::module_& m) {
         R"(openvino_genai.Tokenizer object is used to initialize Tokenizer
            if it's located in a different path than the main model.)")
 
-        .def(py::init([](const std::filesystem::path& tokenizer_path, const std::map<std::string, py::object>& properties) {
+        .def(py::init([](const std::filesystem::path& tokenizer_path, const py::kwargs& kwargs) {
```

Comment:
maybe we need to keep `map` for BW compatibility and add warning similar to `LLMPipeline`?

https://github.com/openvinotoolkit/openvino.genai/blob/bef96dd4103adad9c9c0e33c181bcf16a9fb234b/src/python/py_llm_pipeline.cpp#L110-L116

Reply:
Done.